### PR TITLE
Add CommercialDiscount object

### PR DESCRIPTION
**Pull request by MathiasDL**
Enforcing consistency

### CODE REVIEW

**Review by lorgan3 — DISMISSED** [review]

**Review by joaoinacio — COMMENTED** [review]
👌

**Review by joaoinacio — DISMISSED** [review]

File: src/datastructures.apib [hunk 1]
```diff
@@ -185,3 +185,10 @@
 ## User (object)
 + type: `user` (string)
 + id: `bef8c53b-b6bc-0fde-944e-5c4b16e5155a` (string)
+
+## CommercialDiscount (object)
++ type: `percentage` (enum)
+    + Members
+        + percentage
++ value: 15.5 (number)
```
Comment (tiagobastos) [inline src/datastructures.apib:193]: Q: Useful to keep the label mentioning "Values between 0 and 100"?

**Review by tiagobastos — DISMISSED** [review]

File: src/datastructures.apib (hunk 1 above)
Comment (MathiasDL) [reply src/datastructures.apib:193]: Indeed, I added it again. Although it does not seem to appear on the documentation.. But our next API documentation framework can maybe handle it

Comment (MathiasDL) [reply src/datastructures.apib:193]: Correct, I added it again

**Review by lorgan3 — APPROVED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Comment by ArnaudWeyts** [comment]
✅

**Pull request merged**